packages/client/src/types.ts

```diff
@@ -207,7 +207,9 @@ export interface PolkadotClient {
    *
    * Note that subscribing to this observable already supersedes the need of
    * subscribing to `finalizedBlock$`, since the last element of the array will
-   * be the latest known finalized block.
+   * be the latest known finalized block. There can be gaps between emissions,
+   * i.e. if new best or finalized block jump more than one block only one
+   * emission will be done.
```

Comment:
I don't think that's true - `bestBlocks$` emit an array of all the blocks from the finalized+1 to latest finalized. There are no gaps.

You probably meant to put this to `bestBlock$`?

Reply:
Got it, maybe reword to something like "Note that some blocks might not get reported, specifically if they became finalized immediately without being part of the best-block chain"